lib/date.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+const _ = require('lodash');
+const path = require('path');
+const moment = require('moment');
+const validator = require('hof-form-controller/lib/validation');
```

Comment:
I can't see where this is used, and given it's pulling in all of `hof-form-controller` it would be good to not have it.